packages/atlas/src/components/Avatar/Avatar.tsx

```diff
@@ -38,26 +37,32 @@ export const Avatar: React.FC<AvatarProps> = ({
   size = 'default',
   children,
   onClick,
-  onEditClick,
   className,
   editable,
   newChannel,
   onError,
 }) => {
   const handleEditClick = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
     event.stopPropagation()
-    if (onEditClick) {
-      onEditClick(event)
+    if (onClick) {
+      onClick(event)
     }
   }
   const isEditable = !loading && editable && size !== 'default' && size !== 'bid'
 
   return (
-    <Container onClick={onClick} size={size} className={className} isLoading={loading} withoutOutline={withoutOutline}>
+    <Container
+      onClick={onClick}
+      size={size}
+      className={className}
+      isLoading={loading}
+      withoutOutline={withoutOutline}
+      isClickable={!!onClick}
+    >
       {isEditable && (
         <EditButton size={size} onClick={handleEditClick} type="button">
```

Comment:
I'm wondering if we could simplify this. Do we really need 2 onClick handlers since we get rid of `onEditClick`?

Reply:
I would like to keep `onClick` on button since it may be more natural click target than `div` for screen readers etc

Reply:
Maybe container should be a button then if Avatar is clickable?

Reply:
Yeah that would make more sense, let me refactor that